Bio/Affy/CelFile.py

```diff
@@ -142,61 +158,69 @@ def read4(f):
     record.Algorithm = headersMap["Algorithm"]
     record.AlgorithmParameters = headersMap["AlgorithmParameters"]
     record.NumberCells = preHeadersMap["cellNo"]
-    #record.intensities are set below
-    #record.stdevs are set below
-    #record.npix are set below
+    # record.intensities are set below
+    # record.stdevs are set below
+    # record.npix are set below
     record.nrows = int(headersMap["Rows"])
     record.ncols = int(headersMap["Cols"])
 
-    # These cannot be reliably set in v4, because of discrepancies between real data and the documented format.
+    # These cannot be reliably set in v4, because of discrepancies between real
+    # data and the documented format.
     record.nmask = None
     record.mask = None
     record.noutliers = None
     record.outliers = None
     record.modified = None
 
-    # Real data never seems to have anything but zeros here, but we don't want to take chances. Raising an error is better than returning unreliable data.
+    # Real data never seems to have anything but zeros here, but we don't want
+    # to take chances. Raising an error is better than returning unreliable
+    # data.
 
     if headersMap["Axis-invertX"] != "0":
         raise(ValueError("Parse Error"))
-    
+
     if headersMap["AxisInvertY"] != "0":
         raise(ValueError("Parse Error"))
 
     if headersMap["AxisInvertY"] != "0":
         raise(ValueError("Parse Error"))
 
-    # This is unfortunately undocumented, but it turns out that real data has the `record.AlgorithmParameters` repeated in the data section, until an EOF, i.e. b'\x04'.
+    # This is unfortunately undocumented, but it turns out that real data has
+    # the `record.AlgorithmParameters` repeated in the data section, until an
+    # EOF, i.e. b'\x04'.
     char = b'\x00'
     safetyValve = 10**4
     for i in range(safetyValve):
         char = f.read(1)
         # For debugging
-        #print([i for i in char], end="")
+        # print([i for i in char], end="")
         if char == b'\x04':
             break
         if i == safetyValve:
             raise(ValueError("Parse Error"))
 
     # After that there are precisely 15 bytes padded. Again, undocumented.
     padding = f.read(15)
```

Comment:
Are these fifteen nulls? Should you check, and at least issue a warning if the padding is non-null values? See similar cases in the SFF parser, file ``Bio/SeqIO/SffIO.py``